src/PIL/PngImagePlugin.py

```diff
@@ -630,6 +630,7 @@ class PngImageFile(ImageFile.ImageFile):
 
     format = "PNG"
     format_description = "Portable network graphics"
+    __frame = 0
```

Comment:
really minor, but initializing this in `_open()` would make it more consistent w/the other existing image plugins (I checked  FLI/MPO/TIFF)